frontend/src/app/main/ui/workspace/viewport/actions.cljs

```diff
@@ -263,8 +262,7 @@
  (mf/use-callback
   (fn [event]
     (let [bevent  (.getBrowserEvent ^js event)
-          key     (.-keyCode ^js event)
-          key     (.normalizeKeyCode KeyCodes key)
+          key     (.-key ^js event)
```

Comment:
I don't think you need the `key` anymore.

Reply:
we still need it, because it's read later, in the filter